CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-project(dnfdragora NONE)
+project(dnfdragora C)
```

Comment:
Why are you changing this to a C project? This doesn't make sense, since we don't need to require a C compiler.